packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs-extra';
+const ReadMeContent = `# Getting Started with amplify-cli
+This directory was generated with [amplify-cli][] command. This README helps you to find resources you are looking for.
+
+## About CLI
```

Comment:
Shorten this to 
```
Helpful resources:
- Amplify documentation: docs.amplify.aws
- Amplify CLI documentation: docs.amplify.aws/cli
- More details on this folder & generated files: https://docs.amplify.aws/cli/reference/files
- Join Amplify's Discord community: dicord.gg/amplify
```

Reply:
Thanks for the inputs! For the discord link, I've replaced with community link as it's a parent page of Discord invitation link and there is a possibility to add more pages or chat links for different regions on the community page.

E.g. Users in Japan may not join the Discord because of language barrier, but will join community Slack will be led by community members.